variants/rak_wismeshtap/platformio.ini

```diff
@@ -10,8 +10,14 @@ build_flags = ${nrf52840_base.build_flags} -Ivariants/rak_wismeshtap -D RAK_4631
   -DEINK_WIDTH=250
   -DEINK_HEIGHT=122
   -DMESHTASTIC_EXCLUDE_WIFI=1
-  -DMESHTASTIC_EXCLUDE_EXTERNALNOTIFICATION=1
+  -DMESHTASTIC_EXCLUDE_PAXCOUNTER=1
   -DMESHTASTIC_EXCLUDE_WAYPOINT=1
+  -DMESHTASTIC_EXCLUDE_DETECTIONSENSOR=1
+  -DMESHTASTIC_EXCLUDE_REMOTEHARDWARE=1
+  -DMESHTASTIC_EXCLUDE_AUDIO=1
```

Comment:
These can be removed because they are handled in the parent nrf52.ini:
-DMESHTASTIC_EXCLUDE_PAXCOUNTER=1
-DMESHTASTIC_EXCLUDE_AUDIO=1

Reply:
Thank you. Fixed